to-do-list/src/components/data/exampleList.js

```diff
@@ -0,0 +1,7 @@
+export const task = [
```

Comment:
This folder (data) is not a component, so it should probably be moved up to the `src` folder.